lib/server/push.api.js

```diff
@@ -320,11 +332,14 @@ function serverSend(raixNotification) {
 }
 
 function querySend(query, raixNotification) {
-  console.log('Push: Sending notifications for query: ', query);
+  console.log('Push: Sending notifications for query: ', JSON.stringify(query));
 
   let firebasePromises = [];
 
-  let appTokens = AppTokenUtil.getAppTokensBasedOnQuery(query);
+  let desktopAppId = raixNotification && raixNotification.isDesktop && raixNotification.desktopAppId || undefined;
```

Comment:
@mjovkovic You need to add validation for `isDesktop` and for `desktopAppId` to `packages/push/lib/common/notifications.js` in `_validateDocument` function.

Reply:
@mjovkovic Add validation.

Reply:
👍